src/contrib/running-tests.rst

```diff
@@ -1,24 +1,33 @@
 Running Tests
 =============
 
-SSSD is a complex piece of software with a long development history. Therefore, there are several layers of tests with different goals and using different frameworks. This page shows how to run the tests and how to add new ones or modify the existing tests.
+SSSD is a complex piece of software with a long development history. Therefore, there has been several layers of tests with different goals and using different frameworks. We are currently in progress of reworking all of our tests and standardizing them using our own `SSSD Test Framework <https://tests.sssd.io/en/latest/>`_.
 
-Existing test types
--------------------
+To get started immediately you can visit `Running Tests <https://tests .sssd.io/en/latest/running-tests.html>`_ page in the `SSSD Test Framework <https://tests.sssd.io/en/latest/>`_ wiki and start following the instructions provided.
```

Comment:
`https://tests .sssd.io/en/latest/running-tests.html`

Looks like an extra space after `tests` is here.